### ISSUE TITLE
Number of Total Reaction Stack

### ISSUE THREAD

**Issue by KSUN63** [luost26/ChemProjector#18]
Dear @luost26 ,

Thanks for creating this great repo! I'm trying to get a sense of how many reaction stacks in total were used during the training process. The only relevant text I can find is here: "We use the AdamW optimizer (Loshchilov & Hutter, [2017](https://arxiv.org/html/2406.04628v1#bib.bib33)) to train the network with a learning rate of 3e-4 and a batch size of 256 for 500,000 iterations."

Can I naively assume that the total reaction stacks used are just the product of iterations and the batch size, giving the total reaction to be around 128M?

Also, do you mind if I build some other software on top of your codespace and properly acknowledge you in the GitHub and publication? Thanks!

**Comment by luost26** [comment]
> Can I naively assume that the total reaction stacks used are just the product of iterations and the batch size, giving the total reaction to be around 128M?

Yes, this assumption is correct.

> Also, do you mind if I build some other software on top of your codespace and properly acknowledge you in the GitHub and publication? Thanks!

Please feel free to build your project with this codebase.
